applications/RomApplication/custom_utilities/rom_residuals_utility.h

```diff
@@ -208,6 +220,70 @@ namespace Kratos
         return matrix_residuals;
         }
 
+        Matrix GetProjectedResidualsOntoJPhi(
```

Comment:
Should we substitute all these with a block_for_each ?  I mean the GetResidualsProjectedOnto Phi, Psi, and J Phi ? Instead of keeping raw open mp pragmas?

Reply:
Yes, I am adding this to the TODO list.